corehq/apps/app_manager/models.py

```diff
@@ -4537,6 +4545,16 @@ def get_questions(self, xmlns):
         return questions
 
 
+def domain_has_apps(domain):
```

Comment:
We're trying to move to having all couch view calls in a `dbaccessors.py` file in the package root i.e. `corehq.app_manager.dbaccessors`.